ghost/staff-service/lib/emails.js

```diff
@@ -271,6 +273,17 @@ class StaffServiceEmails {
         return this.mailer.send(msg);
     }
 
+    registerPartials() {
```

Comment:
Added helper to register all partials in `email-templates/partials/` folder (in case we'll add more components in future)